Examples/Sources/MacroExamples/Implementation/Peer/AddCompletionHandlerMacro.swift

```diff
@@ -38,6 +38,7 @@ public struct AddCompletionHandlerMacro: PeerMacro {
       } else {
         newEffects = FunctionEffectSpecifiersSyntax(asyncSpecifier: .keyword(.async))
       }
+      newEffects.asyncSpecifier?.trailingTrivia = [.spaces(1)]
```

Comment:
I think it would be cleaner if you set the trailing trivia directly when creating the `async` keyword.

```swift
        newEffects.asyncSpecifier = .keyword(.async, trailingTrivia: .space)
      } else {
        newEffects = FunctionEffectSpecifiersSyntax(asyncSpecifier: .keyword(.async, trailingTrivia: .space))
      }
```

Reply:
Done